### ISSUE TITLE
Game hangs - computer opponent does not move - after initial moves defined

### ISSUE THREAD

**Issue by CottonEaster** [lukasmonk/lucaschess#50]
Here's the configuration at issue.

![screenshot 271](https://user-images.githubusercontent.com/9998846/34721222-d973e8f2-f539-11e7-9cfb-6e7c2cfdc2a6.png)

![screenshot 272](https://user-images.githubusercontent.com/9998846/34721224-db64adea-f539-11e7-8c04-2fe21bdc8f33.png)

InsLucasChess1104a

**Comment by lukasmonk** [comment]
Yes, Paladin y Hannibal, both doesn´t work with 32 bits PC with old processor. I have to remove.

**Comment by lukasmonk** [comment]
I have recompiled Paladin, to work with old 32 bits processors.
And Hannibal, I have changed to version 1.4b, as it works in 32 bits old.

**Comment by CottonEaster** [comment]
Well, that's good, but: the instance of LucasChess at issue runs on a 64-bit processor (on Windows 8.1). I am sorry I did not supply this information before; I had not realised it would be relevant. I can test the current build for the bug also on a 64-bit processor running LucasChess via Wine on Linux, if that would be helpful.

**Comment by lukasmonk** [comment]
These are the new releases:

[pal_han.zip](https://github.com/lukasmonk/lucaschess/files/1629131/pal_han.zip)

Test please.

You could copy changing the names, to their folders.

**Comment by CottonEaster** [comment]
OK, I've worked out where I should put those files. _But_: (1) these seem to be files for 32-bit systems . . see my comment above; (2) the Paladin file has the suffix '_old' - is it the right file?

**Comment by lukasmonk** [comment]
In this moment I am commiting 11.05 version, with these changes.
Old is right, it is a personal compilation that works in old 32 processors.

**Comment by CottonEaster** [comment]
I'm sorry, Lucas, but I don't understand: will the files you sent me likely solve the problem I've had on my computer, or not?

**Comment by lukasmonk** [comment]
In the new update, those changed files are included with the distribution. Now Paladin from program open the Paladin_32bits_old.exe file, the same in the previous link (pal_han.zip).

**Comment by CottonEaster** [comment]
I am sorry but I still have trouble understanding you. Nevertheless, I replaced my existing files with the two files that you provided. Then I tried to get Lucas Chess to start from the position I was using. I did that using first the Paladin engine and then using the Hannibal engine. In both cases the program hung, i.e. entirely ceased to respond.

**Comment by lukasmonk** [comment]
Are you using the 11.05 version of the program?

**Comment by CottonEaster** [comment]
I'm on version 11.05b now and I think that was the version I had the problems with.

**Comment by lukasmonk** [comment]
Could you exec the programs in the zip ?
Exec + write uci and press enter
